File: docflow-frontend/types/api/OnboardingAuditCursorPageDTO.ts
import { OnboardingAuditRow } from "./OnboardingAuditRow";

export type OnboardingAuditCursorPageDTO = {
  items: OnboardingAuditRow[];
  hasMore: boolean;
  nextCursorTimestamp: string | null;
  nextCursorId: string | null;
};

File: docflow-backend/src/main/java/com/brutecx/docflow_backend/api/dto/audit/OnboardingAuditCursorPageDTO.java
package com.brutecx.docflow_backend.api.dto.audit;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

public record OnboardingAuditCursorPageDTO(
        List<OnboardingAuditDTO> items,
        boolean hasMore,
        Instant nextCursorTimestamp,
        UUID nextCursorId
) implements BaseAuditCursorPageDTO<OnboardingAuditDTO> {
}
